### ISSUE TITLE
Rerender fails

### ISSUE THREAD

**Issue by mrocklin** [martindurant/dask-kubernetes#38]
It looks like there is a typo in the name/cluster variable.  I also get follow on errors about `update_config` after I fix this.

**Comment by martindurant** [comment]
Out of interest, what were you trying to change?
This kind of updating should maybe be removed for the time being, since it probably actually needs "rolling update " by kubernetes to restart containers with new settings to have any effect. Some settings such as cluster region cannot be changed at all.

**Comment by mrocklin** [comment]
I wanted to change a few things all at once and felt that changing the yaml file would be more convenient than using many cli calls.  In particular I wanted to change the node type so that I could get larger worker nodes.  I'm working around this by just making a new cluster.

**Comment by martindurant** [comment]
That brings out exactly the complexity I am talking about. It is possible to [change the instance type](https://cloud.google.com/sdk/gcloud/reference/compute/instances/set-machine-type) , but it involves stopping instances and restarting them, and (presumably) restarting all kubernetes containers too.

**Comment by mrocklin** [comment]
Yeah, I would mostly hope that we just lean on Kubernetes to handle this
